dist/localization.json

```diff
@@ -248,5 +248,103 @@
     "player_vehicle_spawned": "Un(e) {0} est apparue.",
     "player_give": "{0} a reçu {2} {1}.",
     "discord": "Serveur discord: {0}"
-  }
+  },
+	"TH": {
```

Comment:
Indentation is inconsistent with the rest of the file, although I don't think it should be a problem.